### PR TITLE
initial commit for sample app

### PR DESCRIPTION
**Pull request by martin8488**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,5 +28,9 @@ dependencies {
 	compile('org.springframework.boot:spring-boot-starter-actuator')
 	compile('org.springframework.boot:spring-boot-starter-aop')
 	compile('org.springframework.boot:spring-boot-starter-web')
-	testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile('org.springframework.boot:spring-boot-starter-data-jpa')
+	compile(group: 'com.h2database', name: 'h2', version: '1.3.148')
```
Comment (pkissling) [inline build.gradle]: the version should be inherited by the spring boot parent

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    console:
+      enabled: true
+      path: /h2
+  jpa:
+    hibernate:
+      ddl-auto: create
+
+# Datasource
+  datasource:
+    url: jdbc:h2:file:~/test
```
Comment (pkissling) [inline src/main/resources/application.yml]: `jdbc:h2:mem:illustr8` would make sure that the database is in-memory

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    username: sa
+    password:
+    driver-class-name: org.h2.Driver
```
Comment (pkissling) [inline src/main/resources/application.yml]: line 14 - 16 can be omitted, so it will fall back to the default credentials and driver

Comment (martin8488) [reply src/main/resources/application.yml]: patrick ist doof

**Pull request merged**